$OpenBSD: patch-src_video_x11_SDL_x11keyboard_c,v 1.1 2018/10/29 05:39:49 thfr Exp $

fix incorrect function signature that breaks macppc platform (likely any big
endian platform): "stack overflow in function X11_InitKeyboard"
upstream changeset http://hg.libsdl.org/SDL/rev/9dd351b26dcb

Index: src/video/x11/SDL_x11keyboard.c
--- src/video/x11/SDL_x11keyboard.c.orig
+++ src/video/x11/SDL_x11keyboard.c
@@ -266,7 +266,7 @@ X11_InitKeyboard(_THIS)
     int best_distance;
     int best_index;
     int distance;
-    BOOL xkb_repeat = 0;
+    Bool xkb_repeat = 0;

     X11_XAutoRepeatOn(data->display);

